x/gov/client/cli/query.go

```diff
@@ -298,7 +298,7 @@ $ %[1]s query gov votes 1 --page=2 --limit=100
 
 				var votes types.Votes
 				clientCtx.JSONMarshaler.MustUnmarshalJSON(resByTxQuery, &votes)
-				return clientCtx.PrintOutput(votes)
+				return clientCtx.PrintOutputLegacy(votes)
```

Comment:
why legacy here?

Reply:
Because it's not migrated yet. I explained in the PR description

Reply:
see #6987